Makefile

```diff
@@ -115,6 +115,18 @@ release:
 	$(VERSION_CHECK)
 	./scripts/release.sh build-release "$(VERSION_TAG)" "$(BUILD_SYSTEM)" "$(RELEASE_TAGS)" "$(RELEASE_LDFLAGS)"
 
+release-tag:
+	@$(call print, "Adding release tag.")
+
+	@version_file_path="version.go" && \
+	tag=$$(./scripts/get-release-tag-str.sh $$version_file_path); \
```

Comment:
This won't work, since we echo other stuff in the script. Should we just move all the logic into the script and call it `create-release-tag.sh`?

Reply:
Or to re-use the version parsing script, just have a second script that creates the tag itself? Just because the script syntax in the Makefile itself isn't as portable as `bash` I think.

Reply:
I think it does work, I may be misunderstanding what you mean:
```shell
user@laptop:~/dev/taproot-assets$ make release-tag
 Adding release tag.
Reading version fields from version.go
Derived git tag name: v0.5.2
Git tag name version string is SemVer compliant
Adding git tag: v0.5.2
user@laptop:~/dev/taproot-assets$ git log
commit 781d1609316193b246988c5e368dbd96f00e92b5 (HEAD -> validate-release-version, tag: v0.5.2)
```

In the script I write log messages to stderr. Only the git tag name goes into stdout.

Reply:
Oops, I missed the `>&2` indeed. Sorry about that. Looks good to me then.